test/unit/resources/platform_test.rb

```diff
@@ -0,0 +1,97 @@
+# encoding: utf-8
+
+require 'helper'
+require 'inspec/resource'
+
+describe 'Inspec::Resources::Platform' do
+  let(:resource) { resource = MockLoader.new(:ubuntu1504).load_resource('os') }
```

Comment:
Should this be `load_resource('platform')`?

Reply:
or rather, should we actually be doing this twice... once for platform, and once for os?

Reply:
Yeah this should be platform instead of the os subclass.